packages/components/psammead-timestamp/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.11 | [PR#xx](https://github.com/bbc/psammead/pull/xx) Unify package.json, package-lock & changelog versions to prevent Talos borking |
```

Comment:
```suggestion
| 4.0.11 | [PR#4254](https://github.com/bbc/psammead/pull/4254) Unify package.json, package-lock & changelog versions to prevent Talos borking |
```